src/main/scala/com/github/codelionx/distod/types/OrderDependency.scala

```diff
@@ -0,0 +1,37 @@
+package com.github.codelionx.distod.types
+
+
+sealed trait OrderDependency {
+
+  implicit class RichCandidateSet(set: CandidateSet) {
+
+    def toSetString: String = s"{${set.mkString(", ")}}"
+  }
+}
+
+
+object OrderDependency {
+  final case class ConstantOrderDependency(context: CandidateSet, constantAttribute: Int) extends OrderDependency {
+
+    override def toString: String = s"${context.toSetString}: [] ↦ $constantAttribute"
+  }
+
+  final case class EquivalencyOrderDependency(
+      context: CandidateSet,
+      attribute1: Int,
+      attribute2: Int,
+      reverse: Boolean = false
+  ) extends OrderDependency {
+
+    override def toString: String = {
+      val orderOfSecondAttribute =
+        if (reverse)
+          "↓"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://app.codacy.com/app/CodeLionX/distod/pullRequest?prid=4537008)